### ISSUE TITLE
latest version does not capture changelog content

### ISSUE THREAD

**Issue by classabbyamp** [mindsers/changelog-reader-action#63]
Since the latest update (v2.1.0), the contents of the changelog are not captured and outputted by the action.

---

[workflow affected](https://github.com/0x5c/hexchat-void-repos/blob/1319aced3ceb1e0023091afa8dd41bde4cc4c611/.github/workflows/release.yml)

[workflow run](https://github.com/0x5c/hexchat-void-repos/runs/7059293304)

<details><summary>affected workflow step debug log</summary>

```
##[debug]Evaluating condition for step: 'Get Changelog Content'
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: Get Changelog Content
##[debug]Loading inputs
##[debug]Evaluating: env.version_num
##[debug]Evaluating Index:
##[debug]..Evaluating env:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'version_num'
##[debug]=> '1.1.0'
##[debug]Result: '1.1.0'
##[debug]Loading env
Run mindsers/changelog-reader-action@v[2](https://github.com/0x5c/hexchat-void-repos/runs/7059293304?check_suite_focus=true#step:4:2)
::group::Parse data
Parse data
::group::Validate data
Validate data
::set-output name=version::1.1.0
##[debug]steps.changelog_reader.outputs.version='1.1.0'
::set-output name=date::2022-0[6](https://github.com/0x5c/hexchat-void-repos/runs/7059293304?check_suite_focus=true#step:4:6)-26
##[debug]steps.changelog_reader.outputs.date='2022-06-26'
::set-output name=status::released
##[debug]steps.changelog_reader.outputs.status='released'
::set-output name=changes::
##[debug]steps.changelog_reader.outputs.changes=''
##[debug]Node Action run completed with exit code 0
##[debug]Finishing: Get Changelog Content
```

</details>

**Comment by nihartsx** [comment]
Having the same issue, i think it might be related to https://github.com/mindsers/changelog-reader-action/pull/62.

**Comment by anmlnath** [comment]
Also ran into an issue where the changelog changes is no longer being placed in the release. This is across two projects and was working before. That said, I'm not sure if the issue is with changelog-reader-action or with ncipollo/release-action

**Comment by anmlnath** [comment]
I can confirm that specifying `uses: mindsers/changelog-reader-action@v2.0.0` works.

**Comment by classabbyamp** [comment]
> not sure if the issue is with changelog-reader-action or with ncipollo/release-action. 

based on the debug output in my instance, it's definitely changelog-reader, because it shows that the output is an empty string. I also don't use that release action

**Comment by mindsers** [comment]
Hi! Thanks for the report. We're working on it. The exact issue should be fixed by #62
